apps/desktop-wallet/src/pages/UnlockedWallet/AddressesPage/AddressesTabContent.tsx

```diff
@@ -36,6 +123,16 @@ const AddressesTabContent = memo(() => {
       onButtonClick={openNewAddressModal}
       HeaderMiddleComponent={
         <HeaderMiddle>
+          <SelectWrapper>
+            <Select<AddressOrder>
```

Comment:
I didn't even know this syntax is possible 😆 TS should be smart enough to infer the type using the props of the component.